src/feature-test/user_sees_expanded_recipe_when_clicked.test.js

```diff
@@ -0,0 +1,53 @@
+import '../setupTest.js'
+import fetch from '../unit-test/mocks/recipeListFetch';
+
+describe('I.I.A.S', () => {
+  it('the user sees expanded recipe when clicking on title', async () => {
+    const browser = await puppeteer.launch({
+      // slowMo: 40,
+      // This two lines below allows the dev to see the flow on the chromium page
+      // headless: false,
+      // devtools: true
+    });
+    const page = await browser.newPage();
+    await page.goto('http://localhost:3000/');
+    await page.type('input[name="ing1"]', 'potatoes');
```

Comment:
This is the bit where tests are failing.

Reply:
Got it. Didn't realise this had changed on master. All good now.